parachain/tests/adder.rs

```diff
@@ -61,6 +61,7 @@ impl Externalities for DummyExt {
 }
 
 const TEST_CODE: &[u8] = adder::WASM_BINARY;
+const INFINITE_LOOP_CODE: &[u8] = include_bytes!("res/infinite_loop.wasm");
```

Comment:
Where is the source code of this wasm file? Can we not just add an infinite loop to `adder` that is triggered when we give it a certain parameter?

Reply:
The code does nothing, but loops indefinitely. Source code is `loop {}` Using a separate binary seems to me more straightforward than adding obscure behaviour to adder.

Reply:
If at any point we need to recompile it, for whatever reason, could you at least upload the source file and the build script?^^

Reply:
Added as a crate